<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>683</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>963</width>
        <height>82</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>82</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(27, 43, 66);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="HomeButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../PNG/House4.png</normaloff>../PNG/House4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="UserButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../PNG/profile4.png</normaloff>../PNG/profile4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>8</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>118</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(27, 43, 66);
</string>
              </property>
              <property name="text">
               <string>Category</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CategoryComBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>169</width>
                <height>55</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>600</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
	color: rgb(27, 43, 66);
    border: 2px solid  rgb(27, 43, 66);
    border-radius: 10px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #FCFCFC, stop: 0.4 #FAFAFA,
                                 stop: 0.5 #F5F5F5, stop: 1.0 #F2F2F2);
}


QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #F2F2F2, stop: 0.4 #F5F5F5,
                                stop: 0.5 #FAFAFA, stop: 1.0 #FCFCFC);
}

QComboBox:on { 
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
	border-bottom-color: rgb(27, 43, 66);
	border-left-color: rgb(212, 212, 214);
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:on { 
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 2px solid darkgray;
    selection-background-color: white;
}
</string>
              </property>
              <item>
               <property name="text">
                <string>Select category</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>8</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(27, 43, 66);</string>
              </property>
              <property name="text">
               <string>Access to another user</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="UserComBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>169</width>
                <height>55</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>600</width>
                <height>55</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
	color: rgb(27, 43, 66);
    border: 2px solid  rgb(27, 43, 66);
    border-radius: 10px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #FCFCFC, stop: 0.4 #FAFAFA,
                                 stop: 0.5 #F5F5F5, stop: 1.0 #F2F2F2);
}


QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #F2F2F2, stop: 0.4 #F5F5F5,
                                stop: 0.5 #FAFAFA, stop: 1.0 #FCFCFC);
}

QComboBox:on { 
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
	border-bottom-color: rgb(27, 43, 66);
	border-left-color: rgb(212, 212, 214);
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:on { 
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 2px solid white;
    selection-background-color: white;
}
</string>
              </property>
              <item>
               <property name="text">
                <string>Select user</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>132</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(27, 43, 66);</string>
            </property>
            <property name="text">
             <string>Keywords</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="KeywordsTEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>327</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>400</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>1</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>118</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(27, 43, 66);</string>
            </property>
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="TitleLEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>461</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1200</width>
              <height>55</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>118</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(27, 43, 66);
</string>
            </property>
            <property name="text">
             <string>Description </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="DescriptionTEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>461</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1200</width>
              <height>600</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="ExplanationLabel">
      <property name="font">
       <font>
        <family>Yu Gothic Medium</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="CanselButton">
        <property name="minimumSize">
         <size>
          <width>257</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Semilight</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoRepeatDelay">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="SaveNoteButton">
        <property name="minimumSize">
         <size>
          <width>257</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Save Note</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/Vector.png</normaloff>:/new/prefix1/Vector.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
